diff --git a/toolkit/components/extensions/Extension.sys.mjs b/toolkit/components/extensions/Extension.sys.mjs
index 8c4e9f88a1b5..d166e2e08b4 100644
--- a/toolkit/components/extensions/Extension.sys.mjs
+++ b/toolkit/components/extensions/Extension.sys.mjs
@@ -3892,7 +3892,7 @@ export class Extension extends ExtensionData {
       let locales = await this.promiseLocales();

       let matches = Services.locale.negotiateLanguages(
-        Services.locale.appLocalesAsBCP47,
+        ["zh-CN", "zh", ...Services.locale.appLocalesAsBCP47],
         Array.from(locales.keys()),
         this.defaultLocale
       );
